apps/yara/misc_types.h

```diff
@@ -172,6 +172,8 @@ struct YaraFMConfig
 
     // Sparse SA sampling rate.
     static const unsigned SAMPLING =                    10;
+    static const unsigned WORDS_PER_BLOCK =             0;
```

Comment:
Yara (and the old FM-index) used 2 words per block. Please keep binary compatibility. Are you planning to remove (or at least disable) dynamic size?